### PR TITLE
Add option to ignore errors from a specific tag

### PR DESCRIPTION
**Pull request by VincentLanglet**

### CODE REVIEW

File: tests/Integration/IgnoreErrorsIntegrationTest.php [hunk 1]
```diff
@@ -96,5 +96,35 @@ public function invokeDataProvider(): \Generator
             ],
             true,
         ];
+
+        $event = Event::createEvent();
+        $event->setExceptions([new ExceptionDataBag(new \RuntimeException())]);
```
Comment (ste93cry) [inline tests/Integration/IgnoreErrorsIntegrationTest.php:101]: This should be useless for testing purposes

File: tests/Integration/IgnoreErrorsIntegrationTest.php [hunk 2]
```diff
@@ -96,5 +96,35 @@ public function invokeDataProvider(): \Generator
...
+            [
+                'ignore_tags' => [
+                    'route' => 'foo',
+                ],
+            ],
+            true,
+        ];
+
+        $event = Event::createEvent();
+        $event->setExceptions([new ExceptionDataBag(new \RuntimeException())]);
```
Comment (ste93cry) [inline tests/Integration/IgnoreErrorsIntegrationTest.php:116]: This should be useless for testing purposes

**Review by ste93cry — CHANGES_REQUESTED** [review]
Changes look good. Can you please also add a CHANGELOG entry?

**EDIT**
This feature should target the `develop` branch, can you please make the switch and rebase accordingly?

**Comment by VincentLanglet** [comment]
Done @ste93cry :)

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,7 @@
 ## Unreleased
 
 - Allow setting a custom timestamp on the breadcrumbs (#1193)
+- Add option `ignore_tags` to `IgnoreErrorsIntegration` in order to ignore exceptions by tags values.
```
[the remark this answers is not in the document]
Comment (VincentLanglet) [reply CHANGELOG.md]: Done

**Review by Jean85 — APPROVED** [review]

**Review by ste93cry — APPROVED** [review]
Thank you for the contribution 🎉

**Comment by VincentLanglet** [comment]
@ste93cry You're welcome.

What is the release strategy of this package ? :) 
Just to have an idea about how much time I'll have to wait for this feature

**Comment by ste93cry** [comment]
We have no real strategy, we accumulate some things and when we think we have enough features for a new minor version we release it. Sadly, I don't have any ETA for when this may happen though

**Comment by VincentLanglet** [comment]
It's been two month, a new minor release would be really appreciate if it is possible :)

**Comment by ste93cry** [comment]
Release [`3.3.0`](https://github.com/getsentry/sentry-php/releases/tag/3.3.0) is out :ship:

**Pull request merged**